src/media/css/site.styl

```diff
@@ -257,6 +257,12 @@ a:active {
     }    
 }
 
+@media $wide-tablet {
+    .subheader.hidden {
```

Comment:
This two images from (ubuntu) Firefox version 33.0
![screenshot from 2015-07-08 03 38 33](https://cloud.githubusercontent.com/assets/8364578/8568199/b2cfdd0a-2539-11e5-8a99-5b6cb34115d2.png)
![screenshot from 2015-07-08 03 39 03](https://cloud.githubusercontent.com/assets/8364578/8568200/b2d3e18e-2539-11e5-9ca1-68e44a5c7aee.png)